### ISSUE TITLE
Missing argument 1 for FM\\ElFinderPHP\\Connector\\ElFinderConnector::run()

### ISSUE THREAD

**Issue by arlin2050** [helios-ag/ElFinderPHP#22]
Hi,

First of all, thanks for this great connector.
I'm using helios-ag/fm-elfinder-bundle version ~2.3. The bundles helios-ag/fm-elfinder-bundle v2.3.3 and helios-ag/fm-elfinder-php-connector v2.3.1 have been installed with composer. 

It seems that helios-ag/fm-elfinder-php-connector > v2.2.1 isn't compatible with helios-ag/fm-elfinder-bundle v2.3 because of this error : 

```
PHP Warning:  Missing argument 1 for FM\\ElFinderPHP\\Connector\\ElFinderConnector::run(), called in vendor/helios-ag/fm-elfinder-bundle/Loader/ElFinderLoader.php on line 58 and defined in vendor/helios-ag/fm-elfinder-php-connector/src/Connector/ElFinderConnector.php on line 58
```

Any idea ?
Thanks

**Comment by nicolasmure** [comment]
Hi,

You should use the 5.0.5 version of ElFinderBundle who provides the 1st parameter to the `ElFinderConnector::run()` function.

Maybe you could try to edit your composer.json file as following : 
```
"helios-ag/fm-elfinder-bundle": "~5.0",
"helios-ag/fm-elfinder-php-connector": "~2.3"
```
and run `composer update helios-ag/fm-elfinder-bundle helios-ag/fm-elfinder-php-connector`

**Comment by arlin2050** [comment]
This doesn't solved my problem because I use symfony 2.3. I add "helios-ag/fm-elfinder-php-connector": "2.1.*" in my composer.json to solve this problem. An update of the fm-elfinder-bundle dependencies would be great.
